src/storage/store/chunked_node_group.cpp

```diff
@@ -243,6 +243,16 @@ bool ChunkedNodeGroup::delete_(const Transaction* transaction, row_idx_t rowIdxI
     return versionInfo->delete_(transaction, rowIdxInChunk);
 }
 
+void ChunkedNodeGroup::addColumn(Transaction* transaction, TableAddColumnState& addColumnState,
+    bool enableCompression) {
+    auto numRows = getNumRows();
+    auto chunkData = 
+        ColumnChunkFactory::createColumnChunkData(addColumnState.property.getDataType().copy(),
+            enableCompression, capacity, ResidencyState::IN_MEMORY);
```

Comment:
Should also handle the case that the chunked node group is persistent, and we should also flush the appended chunkData to disk.